packages/velo-external-db-core/lib/schema_hooks_utils.js

```diff
@@ -0,0 +1,68 @@
+
+const hooksForAction = (action) => {
```

Comment:
I would rename to `getHooksForAction`.
Alternatively, you can drop this function altogether, export the `HOOKS_FOR_ACTION` const I mentioned in my next comment, and just do `HOOKS_FOR_ACTION[action]`